src/config.rs

```diff
@@ -127,28 +138,26 @@ impl Config {
         }
     }
 
-    pub fn get(&self, key: &str) -> String {
+    pub fn get(&self, key: &str) -> Option<&str> {
```

Comment:
I would prefer panic if the config is not set correctly, Option or Result should be used in runtime error.